### PR TITLE
wait defaults to true

### PR DESCRIPTION
**Pull request by VoR0220**
* Enables waits default to true
* closes eris-ltd/eris-cli#1029

### CODE REVIEW

**Comment by zramsay** [comment]
let's get a deprecation notice for `epm.yaml`s that still have `wait: true` ... epm can still run but should throw a warning by default and tell users they should either eliminate from their `epm.yaml` completely if we don't plan on bringing back the functionality. does epm fail if there's `wait: false`. if not, it should & warn users of the change.

please re-open PR from a fork & additional commit addressing the above comments.

**Comment by VoR0220** [comment]
@zramsay yeah we don't need to worry about it. Errors don't throw for a wait being present even though it does nothing. Will merge.

File: definitions/jobs.go [hunk 1]
```diff
@@ -35,8 +35,6 @@ type Send struct {
 	// (Optional, advanced only) nonce to use when eris-keys signs the transaction (do not use unless you
 	// know what you're doing)
 	Nonce string `mapstructure:"nonce" json:"nonce" yaml:"nonce" toml:"nonce"`
-	// (Optional) wait for the transaction to be confirmed in the blockchain before proceeding
-	Wait bool `mapstructure:"wait" json:"wait" yaml:"wait" toml:"wait"`
```
Comment (benjaminbollen) [inline definitions/jobs.go]: why did you remove this; do we want to disable setting wait to false; if so, where was this discussed?

**Comment by benjaminbollen** [comment]
LGTM
